content/actions/automating-builds-and-tests/building-and-testing-swift.md

```diff
@@ -87,7 +87,7 @@ jobs:
         swift: ["5.2", "5.3"]
     runs-on: {% raw %}${{ matrix.os }}{% endraw %}
     steps:
-      - uses: fwal/setup-swift@ffb5a44dd03d3d22fb26f48fa43e43fa4ce655a7
+      - uses: swift-actions/setup-swift@v1
```

Comment:
We recommend that you pin third party actions to a SHA instead of to a version: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Unless the action is owned by GitHub, we aim to follow this practice in all of our examples. This old sha looks like it does exist on the renamed repo (https://github.com/swift-actions/setup-swift/commit/ffb5a44dd03d3d22fb26f48fa43e43fa4ce655a7), but I'll go ahead and bump the version to match the  sha of the latest release:

```suggestion
      - uses: swift-actions/setup-swift@65540b95f51493d65f5e59e97dcef9629ddf11bf
```